packages/preview/lth-msc-cs/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "lth-msc-cs"
+version = "0.1.0"
+compiler = "0.13.0"
+entrypoint = "lib.typ"
+authors = ["Theodor Lundqvist & Ludvig Delvret"]
+license = "MIT-0"
+description = "A typst version of the offical LaTeX-template for master thesises in computer science at LTH."
+keywords = ["LTH", "Computer Science", "Master Thesis", "Lund University"]
+categories = ["paper"]
+disciplines = ["computer-science", "engineering"]
+
+[template]
+path = "."
```

Comment:
Templates should go in their own sub-directories: only that directory will be copied when creating a project from the template.